src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -33,16 +33,28 @@ public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL)
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_PHONE)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
         }
 
-        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS);
+        Email email = null;
+        if (argMultimap.getValue(PREFIX_EMAIL).isPresent()) {
+            email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
+        } else {
+            email = ParserUtil.parseEmail("filler@filler.com");
+        }
+
+        Address address = null;
+        if (argMultimap.getValue(PREFIX_EMAIL).isPresent()) {
+            address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        } else {
+            address = ParserUtil.parseAddress("Filler Address");
```

Comment:
Why is "Filler Address" here?

Reply:
Can take this as a reply to both comments; when working I realized that deleting the fields breaks something somewhere else in the system and I haven't identified it yet, my workaround to making these fields "optional" wrt user input is to still fill them in with something on the backend so even if the user doesn't enter an email on the front end, it still works.